aten/src/ATen/cuda/CUDABlas.h

```diff
@@ -40,9 +41,9 @@ class PointerModeGuard {
 
 /* LEVEL 3 BLAS FUNCTIONS */
 
-#define CUDABLAS_GEMM_ARGTYPES(Dtype)                                       \
-  char transa, char transb, int64_t m, int64_t n, int64_t k, Dtype alpha,   \
-      const Dtype *a, int64_t lda, const Dtype *b, int64_t ldb, Dtype beta, \
+#define CUDABLAS_GEMM_ARGTYPES(Dtype)                                                               \
+  char transa, char transb, int64_t m, int64_t n, int64_t k, at::acc_type < Dtype , true> alpha,    \
```

Comment:
nit: using `opmath_type` instead of acc_type is more appropriate here (although end effect is the same).